lms/db/__init__.py

```diff
@@ -105,6 +102,7 @@ def init(engine, drop=False, stamp=True):  # pragma: nocover
     try:
         connection.execute(text("select 1 from alembic_version"))
     except sqlalchemy.exc.ProgrammingError:
+        connection.rollback()  # Rollback after failure to find the alembic table
```

Comment:
Why do we have to rollback here? I [didn't have to do that in Via](https://github.com/hypothesis/via/blob/d34733766f9a546b672698ae4c0238578b300c2f/bin/make_db#L12-L21)